tests/test_asb_save_test_result.yml

```diff
@@ -0,0 +1,38 @@
+---
+- hosts: localhost
+  vars:
+    result_file_path: /var/tmp/test-result
+  roles:
+    - ansibleplaybookbundle.asb-modules
+  tasks:
+    - name: prepare test for clean run
+      file:
+        path: "{{ result_file_path }}"
+        state: absent
+
+    - name: encode bind credentials
+      asb_save_test_result:
+        fail: true
+        msg: "Test failed"
+
```

Comment:
Just wondering what the expected/actual behavior is if you do multiple test results.  Will it append new messages to the same file or will it overwrite the original file?  Seems like we'd want multiple test results to show in the file.